ToDoApp/containers/detail/Detail.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import {View, Text, StyleSheet } from 'react-native';
+import ButtonCustom from '../../components/button/ButtonCustom';
+
+class Detail extends Component {
+  static navigationOptions = {
+    title: 'Detail',
+    headerRight: (<View />),
+  }
+  render() {
+    const { navigation } = this.props;
+    return (
+      <View style = {styles.container}>
+        <View style ={{alignItems: 'flex-start'}}>
+            <ButtonCustom text = "Not done"/>
+        </View>
+        <Text style = {styles.primaryText}>{navigation.getParam('first-description')}</Text>
+        <Text style = {styles.secondaryText}>{navigation.getParam('second-description')}</Text>
+        <ButtonCustom text = "MARK AS DONE" />
+      </View>
+    );
+  }
+}
+
+module.exports = Detail;
```

Comment:
please use `export const Detail` instead module.exports

Reply:
I have just changed that, in the next push it will be added.